packages/kbn-rule-data-utils/src/technical_field_names.ts

```diff
@@ -85,6 +85,7 @@ const EVENT_MODULE = 'event.module' as const;
 // Fields pertaining to the alert
 const ALERT_BUILDING_BLOCK_TYPE = `${ALERT_NAMESPACE}.building_block_type` as const;
 const ALERT_EVALUATION_THRESHOLD = `${ALERT_NAMESPACE}.evaluation.threshold` as const;
+const ALERT_EVALUATION_CONDITIONS = `${ALERT_NAMESPACE}.evaluation.conditions` as const;
 const ALERT_EVALUATION_VALUE = `${ALERT_NAMESPACE}.evaluation.value` as const;
```

Comment:
Maybe we can move all the `ALERT_EVALUATION_*` fields to `default_alerts_as_data.`

Reply:
We can, if we want them to apply to all alerts. For any of them that we want to move to apply to all alerts, we'll need to get the security solution team's approval. I'd prefer we just move the one we need for the time being to keep the scope small.